packages/storybook/stories/index.tsx

```diff
@@ -35,6 +35,21 @@ const FeedbackBackground = (
   </svg>
 );
 
+// NOTE(gabro): we're still using createBentoComponents instead of exporting the
+// components directly from @buildo/bento-design-system due to
+// https://github.com/storybookjs/storybook/issues/12185
+// In short, Storybook (which uses react-docgen-typescript) does not pick up the
+// props when the component comes from an external library.
+//
+// The workaround would be to wrap each component in another component that just
+// re-exports it (see https://github.com/storybookjs/storybook/issues/13502#issuecomment-752897475),
+// but that would be way too inconvenient.
+//
+// Using createBentoComponents creates the components directly in Storybook, so
+// their props are picked up correctly.
+//
+// We tried several solutions from the linked issues, but it seems our monorepo
+// setup make this particularly tricky
```

Comment:
@veej I couldn't find a workaround for this, so I've at least documented the whole thing